opengm/plugins/core.py

```diff
@@ -0,0 +1,59 @@
+import re
+import logging
+from pyrogram import Client, filters
+from pyrogram.types import Message, InlineKeyboardMarkup, CallbackQuery, InlineKeyboardButton
+from opengm.opengm import Opengm, command
+from opengm.utils.chat_status import user_admin
+from opengm.utils.plugins import HELPABLE, HELPABLE_LOWER, paginate_plugins
+from opengm.utils.commands import get_args
+# Do not register this plugin!
+
+LOGGER = logging.getLogger(__name__)
+HELPTEXT = """
+Hey there! My name is {}.
+I'm a modular group management bot with a few fun extras! Have a look at the following for an idea of some of the things I can help you with.
+
+**Main** commands avaliable:
+- /start: start the bot
+- /help: PM's this message.
+- /help <module name>: PM's you help for that module.
+"""
+
+
+@Opengm.on_message(filters.command("help"))
+async def help(cl: Client, message: Message) -> None:
+    args = get_args(message)
+    chat = message.chat
+    if chat.type != "private":
+        await message.reply("Contact me in PM to get the list of possible commands.", reply_markup=InlineKeyboardMarkup([[InlineKeyboardButton(text="Help", url="t.me/{}?start=help".format((await cl.get_me()).username))]]))
+    else:
+        if len(args) == 0:
+            keyboard = InlineKeyboardMarkup(inline_keyboard=await paginate_plugins(0, HELPABLE, "help"))
+            await message.reply(HELPTEXT.format("Test"), reply_markup=keyboard)
```

Comment:
I'm aware of the Test here. @thunderbirdtr Can you give me an example on how to get the configuration dict?